spark/src/main/kotlin/com/adevinta/spark/components/rating/RatingLong.kt

```diff
@@ -164,7 +165,7 @@ internal fun RatingFullPreview(
 ) {
     val (theme, userType) = param
     PreviewTheme(theme, userType) {
-        RatingFull(value = 1.6f, commentCount = 0)
```

Comment:
0 is a case that could be usefull to preview too, no ?

Reply:
I changed it to 1 to see the plurals change for Eng, as in french all forms has the same writing "avis", apart from this nothing is changed.